roles/validate_operator_bundle/tasks/main.yml

```diff
@@ -71,8 +71,8 @@
         msg: "Indices: {{ supported_indices['data'] }}"
 
     - name: "Set support_v4_9 to false if there are no supported versions"
-      set_fact:
-        support_v4_9: false
+      fail:
+        msg: "No supported OCP versions found in Pyxis.  Please verify that the version label is valid: {{ ocp_version_annotation }}"
```

Comment:
Cannot see test coverage for the message.
You can get empty list for valid syntax but "ancient" OCP version.

```
> GET /api/containers/v1/operators/indices?sort_by=data.ocp_version&ocp_versions_range==v4.3&organization=redhat-operators HTTP/2
> Host: catalog.redhat.com
> user-agent: curl/7.69.1
> accept: */*
> 
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* old SSL session ID is stale, removing
* Connection state changed (MAX_CONCURRENT_STREAMS == 100)!
< HTTP/2 200 
< server: gunicorn/20.0.4
< content-type: application/json
< content-length: 64
< trace_id: 0xe7db3f8db0cd4be99628e7d605033962
< access-control-allow-origin: *
< cache-control: no-cache, no-store, max-age=0, must-revalidate
< pragma: no-cache
< expires: 0
< x-content-type-options: nosniff
< x-frame-options: DENY
< x-xss-protection: 1 ; mode=block
< referrer-policy: no-referrer
< date: Thu, 16 Sep 2021 14:13:56 GMT
< set-cookie: 377ca4b9aa485619009b75499e9298fd=c2c6d63abc7a07c53af8e6f58ccc83a0; path=/; HttpOnly; Secure
< 
{
  "data": [],
  "page": 0,
  "page_size": 100,
  "total": 0
}
```